content/administration/maintain/mailjet_api.rst

```diff
@@ -0,0 +1,221 @@
+============================
+Set up Mailjet API with Odoo
+============================
```

Comment:
"with odoo" would be in most API doc titles. Same with "set up"

What about
```suggestion
===========
Mailjet API
===========
```

https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#titles-and-headings